### ISSUE TITLE
Could not start operator node in private testnet

### ISSUE THREAD

**Issue by sifnoc** [Onther-Tech/plasma-evm#106]
There is problem for running operator node.

#### System information

Geth version: `geth version`
OS & Version: Windows/Linux/OSX
Commit hash  develop-2cae772f03cc69469f3655c7a0e0cc7b037e42c5

#### Expected behaviour

Run plasma-evm operator well with.

```bash
plasma-evm$ build/bin/geth \
    --nousb \
    --datadir ./chaindata-oper \
    --syncmode='full' \
    --networkid 16 \
    --rootchain.url ws://localhost:8546 \
    --operator 0x71562b71999873DB5b286dF957af199Ec94617F7 \
    --port 30306 \
    --nat extip:::1 \
    --maxpeers 50 \
    --unlock 0x71562b71999873DB5b286dF957af199Ec94617F7 \
    --password signer.pass \
    --nodekeyhex e854e2f029be6364f0f961bd7571fd4431f99355b51ab79d23c56506f5f1a7c3 \
    --mine \
    --miner.gastarget 7500000 \
    --miner.gaslimit 10000000
```

And also tried modified command for new commit.

```bash
plasma-evm$ build/bin/geth \
    --nousb \
    --datadir ./chaindata-oper \
    --syncmode='full' \
    --networkid 16 \
    --rootchain.url ws://localhost:8546 \
    --unlock 0x71562b71999873DB5b286dF957af199Ec94617F7 \
    --password signer.pass \
    --rootchain.sender 0x71562b71999873DB5b286dF957af199Ec94617F7
    --port 30306 \
    --nat extip:::1 \
    --maxpeers 50 \
    --nodekeyhex e854e2f029be6364f0f961bd7571fd4431f99355b51ab79d23c56506f5f1a7c3 \
    --mine \
    --miner.gastarget 7500000 \
    --miner.gaslimit 10000000
```

#### Actual behaviour

Both Panic

#### Steps to reproduce the behaviour

As followed, step 1 to 5 in [docs.tokamak.network - setup-operator-node](https://docs.tokamak.network/docs/en/guides/getting-started/how-to-open-private-testnet-manually#setup-operator-node)

Not using any stamina flags.

#### Backtrace

````text
[genesis.json]

{
  "config": {
    "chainId": 16,
    "homesteadBlock": 0,
    "eip150Block": 0,
    "eip150Hash": "0x0000000000000000000000000000000000000000000000000000000000000000",
    "eip155Block": 0,
    "eip158Block": 0,
    "byzantiumBlock": 0,
    "constantinopleBlock": 0,
    "petersburgBlock": 0,
    "istanbulBlock": 0,
    "muirGlacierBlock": 0,
    "ethash": {},
    "Stamina": {
      "initialized": true,
      "operatorAmount": 1000000000000000000,
      "minDeposit": 500000000000000000,
      "recoveryEpochLength": 120960,
      "withdrarwalDelay": 362880
    }
  },
  "nonce": "0x0",
  "timestamp": "0x0",
  "extraData": "0x1acdf9985559919b7de44d86a6faf7852a62f146",
  "gasLimit": "0x5f5e100",
  "difficulty": "0x1",
  "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000",
  "coinbase": "0x0000000000000000000000000000000000000000",
  "alloc": {
    "0000000000000000000000000000000000000001": {
      "balance": "0x1"
    },
    "0000000000000000000000000000000000000002": {
      "balance": "0x1"
    },
    "0000000000000000000000000000000000000003": {
      "balance": "0x1"
    },
    "0000000000000000000000000000000000000004": {
      "balance": "0x1"
    },
    "0000000000000000000000000000000000000005": {
      "balance": "0x1"
    },
    "0000000000000000000000000000000000000006": {
      "balance": "0x1"
    },
    "0000000000000000000000000000000000000007": {
      "balance": "0x1"
    },
    "0000000000000000000000000000000000000008": {
      "balance": "0x1"
    },
    "000000000000000000000000000000000000dead": {
      "code": "<too long to pasted>",
      "storage": {
        "0x0000000000000000000000000000000000000000000000000000000000000008": "0x0000000000000000000000000000000000000000000000000000000000000001",
        "0x0000000000000000000000000000000000000000000000000000000000000009": "0x00000000000000000000000000000000000000000000000006f05b59d3b20000",
        "0x000000000000000000000000000000000000000000000000000000000000000a": "0x000000000000000000000000000000000000000000000000000000000001d880",
        "0x000000000000000000000000000000000000000000000000000000000000000b": "0x0000000000000000000000000000000000000000000000000000000000058980",
        "0x78b708b837682e6270dbef34cca5b6d3a3fb2b6c6b8bd38a36a62bed9cf03033": "0x00000000000000000000000071562b71999873db5b286df957af199ec94617f7",
        "0x9541d803110b392ecde8e03af7ae34d4457eb4934dac09903ccee819bec4a355": "0x0000000000000000000000000000000000000000000000000de0b6b3a7640000"
      },
      "balance": "0x0"
    },
    "71562b71999873db5b286df957af199ec94617f7": {
      "balance": "0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff7"
    }
  },
  "number": "0x0",
  "gasUsed": "0x0",
  "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000"
}


[backtrace]
plasma-evm $ run.oper.sh

INFO [03-24|17:57:35.567] Maximum peer count                       ETH=50 LES=0 total=50
INFO [03-24|17:57:35.576] Set options for submitting a block       mingaspirce=1000000000 maxgasprice=100000000000 resubmit=10s
INFO [03-24|17:57:35.576] Starting peer-to-peer node               instance=Geth/v1.9.10-stable-2cae772f/darwin-amd64/go1.13
INFO [03-24|17:57:35.576] Allocated trie memory caches             clean=256.00MiB dirty=256.00MiB
INFO [03-24|17:57:35.577] Allocated cache and file handles         database=/Users/jinhwan/.gvm/pkgsets/go1.13/global/src/github.com/Onther-Tech/plasma-evm/chaindata-oper/geth/chaindata cache=512.00MiB handles=5120
INFO [03-24|17:57:37.032] Opened ancient database                  database=/Users/jinhwan/.gvm/pkgsets/go1.13/global/src/github.com/Onther-Tech/plasma-evm/chaindata-oper/geth/chaindata/ancient
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x4957758]

goroutine 1 [running]:
github.com/Onther-Tech/plasma-evm/pls.New(0xc0003d1c60, 0xc0000cbc00, 0x4ee6800, 0xc000675808, 0x400f438)
        /Users/jinhwan/.gvm/pkgsets/go1.13/global/src/github.com/Onther-Tech/plasma-evm/pls/backend.go:143 +0x318
github.com/Onther-Tech/plasma-evm/cmd/utils.RegisterPlsService.func2(0xc0003d1c60, 0xc0002bfa70, 0xc0006759b8, 0xc0003d1c20, 0x2)
        /Users/jinhwan/.gvm/pkgsets/go1.13/global/src/github.com/Onther-Tech/plasma-evm/cmd/utils/flags.go:1878 +0x3d
github.com/Onther-Tech/plasma-evm/node.(*Node).Start(0xc0002cca00, 0x0, 0x0)
        /Users/jinhwan/.gvm/pkgsets/go1.13/global/src/github.com/Onther-Tech/plasma-evm/node/node.go:206 +0x408
github.com/Onther-Tech/plasma-evm/cmd/utils.StartNode(0xc0002cca00)
        /Users/jinhwan/.gvm/pkgsets/go1.13/global/src/github.com/Onther-Tech/plasma-evm/cmd/utils/cmd.go:67 +0x2f
main.startNode(0xc00011e000, 0xc0002cca00)
        /Users/jinhwan/.gvm/pkgsets/go1.13/global/src/github.com/Onther-Tech/plasma-evm/cmd/geth/main.go:359 +0x70
main.geth(0xc00011e000, 0x0, 0x0)
        /Users/jinhwan/.gvm/pkgsets/go1.13/global/src/github.com/Onther-Tech/plasma-evm/cmd/geth/main.go:347 +0xc8
gopkg.in/urfave/cli%2ev1.HandleAction(0x4d54740, 0x51584b8, 0xc00011e000, 0xc0000da360, 0x0)
        /Users/jinhwan/.gvm/pkgsets/go1.13/global/pkg/mod/gopkg.in/urfave/cli.v1@v1.20.0/app.go:490 +0xc8
gopkg.in/urfave/cli%2ev1.(*App).Run(0xc0000c6000, 0xc0000e2000, 0x1b, 0x1c, 0x0, 0x0)
        /Users/jinhwan/.gvm/pkgsets/go1.13/global/pkg/mod/gopkg.in/urfave/cli.v1@v1.20.0/app.go:264 +0x58c
main.main()
        /Users/jinhwan/.gvm/pkgsets/go1.13/global/src/github.com/Onther-Tech/plasma-evm/cmd/geth/main.go:287 +0x55

````

**Comment by 4000D** [comment]
When running operator node, only use `operator`, `operator.password` flags. `unlock` and `password` flags are quite different in the context of running operator node.

**Comment by 4000D** [comment]
I think the problem is `genesis.config.Stamina`. I recommend to change it to `genesis.config.stamina`

**Comment by 4000D** [comment]
fixed at 11c34d672d57a72fa2a4cd6a049e0a59ca26a0f6 and 2d949c6f5e40c82e3e599af9a1bcfd4f1360a8ce
